pkg/pubsub/publisher/publisher.go

```diff
@@ -73,7 +74,7 @@ func NewPublisher(ctx context.Context, inbound HttpMessageReceiver, topic *pubsu
 
 // Start blocks to receive events over HTTP.
 func (p *Publisher) Start(ctx context.Context) error {
-	return p.inbound.StartListen(ctx, p)
+	return p.inbound.StartListen(ctx, kncloudevents.CreateHandler(p))
```

Comment:
Just from quick tracking, it looks like this is double wrapped too?   I think inbound is created via InitializePublisher(), which calls gcp/pkg/utils/clients.NewHTTPMessageReceiver, which in turn calls eventing/pkg/kncloudevents/NewHttpMessageReceiver(), which returns a kncloudevents.HttpMessageReceiver that has a call to CreateHandler() already inside it's StartListen handler?

Reply:
I was just looking at the same thing. I think we should just get B3 handling for free since it was added to the kncloudevents message receiver.

Reply:
Yes it did, thanks!